### PR TITLE
Chore(readme): visual rework in meteor reame

### PR DESCRIPTION
**Pull request by victoriaquasar**
## Summary

- Reworked the core repository `README.MD` to provide cleaner information about the framework, easier naviagion and user experience

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,63 @@
...
+<hr>
 
-Run it:
+# 🔥 Getting Started
 
-```bash
-cd my-app
-meteor
-```
+What about to try a getting started tutorial in your favorite technology?
```
Comment (Grubba27) [inline README.md:48]: I would keep this part of creating and running it 


## Quick Start

On your platform, use this line:

```sh
> npm install -g meteor
```

🚀 To create a project:

```sh
> meteor create my-app
```

☄️ Run it:

```sh
> cd my-app
> meteor
```

**Review by Grubba27 — CHANGES_REQUESTED** [review]
Looks awesome!!! I guess I would make that change and we are rocking! What do you think 
@fredmaiaarantes ?

**Comment by Grubba27** [comment]
For those who wants to see how would look like this is a print of the new readme 

Captura de Tela 2022-08-11 às 18 57 32

File: README.md (hunk 1 above)
Comment (victoriaquasar) [reply README.md:48]: Solved in 2c5bbb05249d3f2d68be097a340328bc521d4620

**Review by Grubba27 — APPROVED** [review]
LGTM!

**Review by fredmaiaarantes — APPROVED** [review]

**Comment by fredmaiaarantes** [comment]
> Looks awesome!!! I guess I would make that change and we are rocking! What do you think @fredmaiaarantes ?

I think it's a great improvement!

**Pull request merged**